### PR TITLE
feat(route): hackmd

### PR DESCRIPTION
**Pull request by kaiix**
## 该 PR 相关 Issue / Involved issue

Close #4688

## 完整路由地址 / Example for the proposed route(s)

```routes
/hackmd/profile/hackmd
```

## 新 RSS 检查列表 / New RSS Script Checklist

- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

### CODE REVIEW

File: lib/v2/hackmd/profile.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    const path = ctx.params.path;
+
+    const response = await got({
+        method: 'get',
+        url: `https://hackmd.io/api/@${path}/overview`,
+    });
+    const data = response.data;
+
+    const profile = data.user || data.team;
+    const notes = data.notes.sort((a, b) => Date.parse(a.createdAt) - b.createdAt);
```
Comment (TonyRL) [inline lib/v2/hackmd/profile.js:13]: No need to sort them.
RSSHub will sort the items automatically
https://github.com/DIYgod/RSSHub/blob/729a16cf83c8651cde5e80fa50e6cb4264ce9ee5/lib/middleware/parameter.js#L58

File: lib/v2/hackmd/profile.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    ctx.state.data = {
+        title: `${profile.name}'s Profile`,
+        link: `https://hackmd.io/@${path}`,
+        description: `${profile.name}'s profile on HackMD`,
+        item: notes.map((note) => ({
+            title: note.title,
+            description: `<pre>${note.content}</pre>`,
```
Comment (TonyRL) [inline lib/v2/hackmd/profile.js:21]: Since `note.content` contains markdown content, it would be better to render the content

**Review by TonyRL — COMMENTED** [review]
Missing `maintainer.js`

File: lib/v2/hackmd/profile.js (hunk 2 above)
Comment (kaiix) [reply lib/v2/hackmd/profile.js:19]: I've checked the value of `note.content`, it's pure text without any markup.

Comment (TonyRL) [reply lib/v2/hackmd/profile.js:19]: Please double-check it with the user `@emisjerry`

Comment (kaiix) [reply lib/v2/hackmd/profile.js:19]: You can check it on the origin website (https://hackmd.io/@emisjerry), it shows the raw markdown in the profile page, not using the rendered content

image

**Pull request merged**